### DISCUSSION TITLE
Nombre de Este Elemento

### DISCUSSION THREAD

**Discussion by oscarflch** [Launch-X-Latam/MisionFrontEnd#283] [Q&A]
Buenas Noches, tengo una duda, este elemento tiene algún nombre en particular? o es solo un rectángulo flotante.
![Banner](https://user-images.githubusercontent.com/114228774/198942069-cd5320da-62fc-4bf3-8ee3-3771e8f871f3.PNG)

**Comment by Amhedriel** [comment]
Buenas, ahí veo muchos elementos, asi que si dices cuadrado flotante te refieres al `<button>`? o si será un `input ="button"` o `input="submit"` pero me parece que no es un formulario y este último no me convence tanto.

**Comment by doguedogue** [comment]
es un div con una posición absoluta 

https://www.w3schools.com/Css/css_positioning.asp

**Comment by fabiancgc12** [comment]
Es un div flotante, te recomiendo que al elemento padre le coloques position: relative y a este elemento lo hagas absolute. A partir de alli ya te será mas fácil el posicionarlo
